Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -237,7 +779,7 @@ private void RenderConfigurations()
             }
         }
 
-        private void ApplyConfigurations()
+        public void ApplyConfigurations()
```

Comment:
Does this need to be changed to `public`? I couldn't find it referenced anywhere outside this class